ddtrace/internal/import_hooks.py

```diff
@@ -51,6 +51,12 @@
 """
 import sys
 import threading
+from types import ModuleType
```

Comment:
See https://github.com/DataDog/dd-trace-py/pull/2256 which removes this module entirely